bin/node/runtime/src/lib.rs

```diff
@@ -549,6 +555,16 @@ impl pallet_election_provider_multi_phase::Config for Runtime {
 	type MinerMaxWeight = MinerMaxWeight;
 	type MinerMaxLength = MinerMaxLength;
 	type MinerTxPriority = MultiPhaseUnsignedPriority;
+	type SignedMaxSubmissions = SignedMaxSubmissions;
+	type SignedRewardBase = SignedRewardBase;
+	type SignedRewardFactor = (); // no score-based reward
+	type SignedRewardMax = SignedRewardBase;
+	type SignedDepositBase = SignedDepositBase;
+	type SignedDepositByte = SignedDepositByte;
+	type SignedDepositWeight = ();
+	type SignedMaxWeight = MinerMaxWeight;
+	type SlashHandler = (); // burn slashes
+	type RewardHandler = (); // nothing to do upon rewards
```

Comment:
I'm not ever sure what this is.

Reply:
It appears to be a hook that gets called if there's a positive imbalance resulting from depositing the reward to the author when a solution is accepted. I don't think it's causing any harm.

Reply:
I'm not quite sure what I had in mind to be even done with the `positive_imbalance`. The big question is, when we reward someone, where does the money come from. Currently, we make that out of thin air (mint it). Maybe later on this hook can be used to burn the money elsewhere. But I don't think this patter is really used anywhere. 

Ideally, this pallet needs to maintain its own internal pot, and have a self-sustaining economy. For now, it is probably okay.